### ISSUE TITLE
Can't assign blank aria-label to md-icon

### ISSUE THREAD

**Issue by wallace41290** [angular/material2#3358]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
If I create an <md-icon aria-label="">person_outline</md-icon>, the aria-label attribute should be present and empty. Perhaps it can provide this functionality when you also supply the role="presentation"

#### What is the current behavior?
If I create an <md-icon aria-label="">person_outline</md-icon>, the aria-label attribute becomes aria-label="person_outline"

Also you can't set the role like role="presentation", you have to do [attr.role]="'presentation'". Otherwise it turns into role="img"

#### What are the steps to reproduce?
As I described above

Providing a Plunker (or similar) is the *best* way to get the team to see your issue.
http://plnkr.co/edit/DXVsLKhyrc5zpWFRoF8v?p=preview

#### What is the use-case or motivation for changing an existing behavior?
If I have an icon that is for presentation only and I don't want the screen reader to read it, I should be able to put a role="presentation" and/or aria-label="" to achieve this.

#### Which versions of Angular, Material, OS, browsers are affected?
beta.1, beta.2

#### Is there anything else we should know?
no

**Comment by jelbourn** [comment]
md-icon no longer sets an aria label to itself and will now always defer to the user-provided label
